src/actions/transactionBuilder.js

```diff
@@ -82,3 +82,23 @@ export function fetchSequence(accountId, horizonBaseUrl) {
       .catch(r => dispatch({type: FETCH_SEQUENCE_FAIL, payload: r}))
   }
 }
+
+
+export const FETCH_BASE_FEE = 'FETCH_BASE_FEE';
+export const FETCH_BASE_FEE_FAIL = 'FETCH_BASE_FEE_FAIL';
+export const FETCH_BASE_FEE_SUCCESS = 'FETCH_BASE_FEE_SUCCESS';
+export function fetchBaseFee(horizonBaseUrl) {
+  return dispatch => {
+    dispatch({
+      type: FETCH_BASE_FEE
+    });
+    axios.get(horizonBaseUrl + '/fee_stats')
+      .then(r => dispatch({
+        type: FETCH_BASE_FEE_SUCCESS,
+        base_fee: r.data.last_ledger_base_fee
```

Comment:
I'm not sure if this is the one we want, @accordeiro I believe this might be the "base fee" i.e. the defined constant? I'm not sure though. It might be better to use `fee_charged.mode` instead

Reply:
Yep, good catch! `fee_charged.mode` seems more accurate.